frontend/src/components/molecules/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import { styled, alpha } from '@mui/material/styles';
+import AppBar from '@mui/material/AppBar';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ISPP-Eventia/eventus/pullRequest?prid=9096486)
```suggestion
import AppBar from "@mui/material/AppBar";
```